api/src/main/java/org/hawkular/apm/api/model/trace/Node.java

```diff
@@ -324,16 +348,18 @@ public void addInteractionId(String id) {
      * @param scope The scope
      * @return The subset of correlation ids that are associated with the scope
      */
-    public List<CorrelationIdentifier> getCorrelationIds(Scope scope) {
+    public List<CorrelationIdentifier> findCorrelationIds(Scope... scope) {
```

Comment:
Haven't actually tested it, but think it will be more efficient that creating a set, just passing in the array and working through it - rather than also creating an iterator (if set).